packages/@aws-cdk/custom-resource-handlers/lib/aws-stepfunctions-tasks/cross-region-aws-sdk-handler/index.ts

```diff
@@ -0,0 +1,79 @@
+/* eslint-disable @typescript-eslint/no-require-imports */
+/* eslint-disable no-console */
+
+type Event = {
+  /**
+   * AWS Region name to call the service
+   */
+  region: string;
+
+  /**
+   * AWS service name (must be in AWS SDK for JS v3 style)
+   */
+  service: string;
+
+  /**
+   * AWS API endpoint
+   */
+  endpoint: string;
+
+  /**
+   * API Action name (must be in camelCase or PascalCase)
+   */
+  action: string;
+
+  /**
+   * request parameters to call the API (must be in AWS SDK for JS v3 style)
+   */
+  parameters?: any;
+};
+
+// reference: code from CDK AwsCustomResource
+function findV3ClientConstructor(pkg: object) {
+  const [_clientName, ServiceClient] = Object.entries(pkg).find(([name]) => {
```

Comment:
It might be a good idea to `filter` the entries instead, and throw if `length !== 1`, just in case multiple `Client`s are present in a single package